### ISSUE TITLE
Document `accepted_field_type`

### ISSUE THREAD

**Issue by ericphanson** [beacon-biosignals/Legolas.jl#60]
IMO this should be documented (docstring + entry in docs), since it is mentioned in the tour, and the uses in e.g. https://github.com/beacon-biosignals/Onda.jl/pull/133/files?diff=split&w=0#diff-5b453f8f7fb34afbebfc6f688a8209aa0532c8b1c3e95393f97afcbc37a473e7R59 look sufficiently generic that I think it will be useful for other users too.

**Comment by jrevels** [comment]
Yup, this is definitely part of Legolas' API (i.e. if it changes in a breaking manner, that's a breaking change for the package) - wouldn't include it in the tour otherwise :) 

will make a PR to add the missing docstring
